integration-libs/epd-visualization/components/visual-viewer/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './models/index';
+export * from './toolbar/index';
+export * from './visual-viewer.component';
```

Comment:
I discovered only now that we don't export `visual-viewer.service`. Was it intentional? Was it due to unstable types it depends on or because of a different reasons?

Reply:
That was not an intended omission. I will update it.

Reply:
I have added the missing export statement.